code/src/components/subintroElement/SubintroElement.tsx

```diff
@@ -0,0 +1,22 @@
+import Box from "@mui/material/Box";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+type SubintroElement = {
```

Comment:
We use convention for types. Our props should be called `ComponentName` + `Props`, in your case `SubintroElementProps`

Reply:
Done